openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -245,13 +245,22 @@
 - control_key: IA-4 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
```

Comment:
```suggestion
  implementation_status: not applicable
```

Reply:
@kevindoingbizas With login creation deferred to the IdP, what do you think of marking this as ``not applicable``?